machine/dell/dell_s4000_c2338/machine.make

```diff
@@ -0,0 +1,34 @@
+# DELL S4000 Series
+# CPU Module: Intel Atom Rangeley (C2338)
+
+ONIE_ARCH ?= x86_64
+
+VENDOR_REV ?= 0
+
+# Translate hardware revision to ONIE hardware revision
+ifeq ($(VENDOR_REV),0)
+  MACHINE_REV = 0
+else
+  $(warning Unknown VENDOR_REV '$(VENDOR_REV)' for MACHINE '$(MACHINE)')
+  $(error Unknown VENDOR_REV)
+endif
+
+# Vendor ID -- IANA Private Enterprise Number:
+# http://www.iana.org/assignments/enterprise-numbers
+# DELL IANA number
+VENDOR_ID = 674
+
+# This platform requires the PXE_EFI64 installer
+PXE_EFI64_ENABLE = yes
+
+CONSOLE_SPEED = 115200
+CONSOLE_DEV = 0
+CONSOLE_FLAG = 0
+
+EXTRA_CMDLINE_LINUX = i2c-ismt.bus_speed=100
```

Comment:
You are the hardware vendor.  You are saying there is a technical reason to use 100 on S4000 or are you saying you are just using 100 because that is what DELL does elsewhere?  I'm trying to understand the technical reason.